sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -72,7 +72,7 @@ PRINT_CONFIG_VAR(VIEWVIDEO_RTP_TIME_INC)
 #ifdef VIDEO_THREAD_SHOT_PATH
 #define VIEWVIDEO_SHOT_PATH VIDEO_THREAD_SHOT_PATH
 #else
-#define VIEWVIDEO_SHOT_PATH /data/video/images
+#define VIEWVIDEO_SHOT_PATH /data/ftp/internal_000/images
```

Comment:
maybe we can just remove the entire VIEWVIDEO_SHOT_PATH thing. I don't think we need to create those streamer files and it's not used for anything else. Also, VIDEO_THREAD_SHOT_PATH has already been long gone; video_capture.c is the boss now.